Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/Batching/Batching.xaml.cs

```diff
@@ -0,0 +1,240 @@
+            var catFile = StorageFile.GetFileFromApplicationUriAsync(new Uri("ms-appx:///InputData/kitten.png")).GetAwaiter().GetResult();
+            var birdImage = CreateSoftwareBitmapFromStorageFile(birdFile);
+            var catImage = CreateSoftwareBitmapFromStorageFile(catFile);
+            var inputImages = new List<VideoFrame>();
+            for (int i = 0; i < 25; i++)
+            {
+                inputImages.Add(VideoFrame.CreateWithSoftwareBitmap(birdImage));
+                inputImages.Add(VideoFrame.CreateWithSoftwareBitmap(catImage));
+            }
+            return inputImages;
+        }
+
+        private SoftwareBitmap CreateSoftwareBitmapFromStorageFile(StorageFile file)
+        {
+            var stream = file.OpenAsync(FileAccessMode.Read).GetAwaiter().GetResult();
+            var decoder = BitmapDecoder.CreateAsync(stream).GetAwaiter().GetResult();
+            return decoder.GetSoftwareBitmapAsync().GetAwaiter().GetResult();
+        }
+
+        private void UpdateEvalText(bool isBatchingEval)
+        {
+            if (isBatchingEval)
+                EvalText.Text = "Inferencing Batched Inputs:";
+            else
+                EvalText.Text = "Inferencing Non-Batched Inputs:";
+        }
+
+        private void CreateSessions(int batchSizeOverride)
+        {
+            var modelPath = "ms-appx:///Models/squeezenet1.1-7.onnx";
+            nonBatchingSession_ = CreateLearningModelSession(modelPath);
+            BatchingSession_ = CreateLearningModelSession(modelPath, batchSizeOverride);
+        }
+
+        private LearningModelSession CreateLearningModelSession(string modelPath, int batchSizeOverride=-1)
+        {
+            var model = CreateLearningModel(modelPath);
+            var deviceKind =
+                (DeviceComboBox.SelectedIndex == 0) ?
+                    LearningModelDeviceKind.Cpu :
+                    LearningModelDeviceKind.DirectXHighPerformance;
+            var device = new LearningModelDevice(deviceKind);
+            var options = new LearningModelSessionOptions()
+            {
+                CloseModelOnSessionCreation = true // Close the model to prevent extra memory usage
+            };
+            if (batchSizeOverride > 0)
+            {
+                options.BatchSizeOverride = (uint)batchSizeOverride;
+            }
+            var session = new LearningModelSession(model, device, options);
+            return session;
+        }
+
+        private static LearningModel CreateLearningModel(string modelPath)
+        {
+            var uri = new Uri(modelPath);
+            var file = StorageFile.GetFileFromApplicationUriAsync(uri).GetAwaiter().GetResult();
+            return LearningModel.LoadFromStorageFileAsync(file).GetAwaiter().GetResult();
+        }
+
+        async private System.Threading.Tasks.Task Classify(List<VideoFrame> inputImages)
+        {
+            float totalEvalDurations = 0;
+            for (int i = 0; i < 100; i++)
```

Comment:
Is there a way to avoid using magic number here? can we make 100 configurable somehow?

Reply:
Done.